### ISSUE TITLE
Android build failure on PhoneGap Build

### ISSUE THREAD

**Issue by MHeasell** [intercom/intercom-cordova#71]
## Version info
  - Intercom for Cordova/PhoneGap version: 3.0.6
  - iOS/Android OS version: N/A
  - iOS/Android Device:  N/A

## Logs
[build-log.txt](https://github.com/intercom/intercom-cordova/files/420061/build-log.txt)

## Details

I am upgrading to version 3.0.6 from 1.1.7. My app fails to build on Android when I specify version 3.0.6 in my plugin entry in config.xml.

I have attached the full build log above. Here are the relevant lines:

```
:compileDebugJavaWithJavac
:compileDebugJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/project/src/io/intercom/android/sdk/IntercomBridge.java:24: error: cannot find symbol
import io.intercom.android.sdk.Gcm;
                              ^
  symbol:   class Gcm
  location: package io.intercom.android.sdk
/project/src/io/intercom/android/sdk/IntercomBridge.java:69: error: cannot find symbol
                Gcm.cacheSenderId(context, senderId);
                ^
  symbol:   variable Gcm
  location: class IntercomBridge
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors
:compileDebugJavaWithJavac FAILED
```

Is this an issue with intercom-cordova or just with my setup? Any help is greatly appreciated.

**Comment by DaleCantwell** [comment]
Hi @MHeasell. This issue is on our side. I'm fixing it now. I'll contact you as soon as the fix is live.
Thanks for reporting the issue to us.

**Comment by DaleCantwell** [comment]
This issue is now fixed in [3.0.7](https://github.com/intercom/intercom-cordova/releases/tag/3.0.7) 
Thanks again for getting in touch with us.

**Comment by MHeasell** [comment]
@DaleCantwell Fantastic. I can confirm that my issue is fixed in 3.0.7, thanks very much for the speedy patch.
